File: MT.Model/Business/ProjectWorkshop.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region ProjectWorkshop:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a ProjectWorkshop:EntityBase.
    /// </summary>
    public class ProjectWorkshop : EntityBase
    {
        [Key]
        public Guid ProjectWorkshopID { get; set; }
        public Guid ProjectID { get; set; }
        public DateTime Date { get; set; }
        public DateTime Time { get; set; }
        public string Adderess { get; set; }
        public string Contents { get; set; }
        public int SortOrder { get; set; }

        /// <summary>
        /// Danh sách file đính kèm
        /// </summary>
        /// Create by: manh:24.02.2018
        public List<ProjectWorkshop_AttachDetail> Details { get; set; }

        public string IdFiles { get; set; }
    }
    #endregion
}

File: BECore/app/model/ProjectWorkshop.js
/// <summary>
/// Model EXTJS ProjectWorkshop.
/// </summary>
Ext.define('QLDT.model.ProjectWorkshop', {
    extend: 'Ext.data.Model',
    idProperty: 'ProjectWorkshopID',
    idType: 'Guid',
    fields: [
        { name: "ProjectWorkshopID", type: "string" },
		{ name: "ProjectID", type: "string" },
		{ name: "Date", type: "date" },
		{ name: "Time", type: "date" },
		{ name: "Adderess", type: "string" },
		{ name: "Contents", type: "string" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
        { name: 'IdFiles',type:'string' }
    ]
});

